apps/autoscheduler/AutoSchedule.cpp

```diff
@@ -2389,22 +2424,22 @@ struct State {
             // We don't need to schedule nodes that represent inputs,
             // and there are no other decisions to be made about them
             // at this time.
-            // aslog(0) << "Skipping over scheduling input node: " << node->func.name() << "\n";
+            // aslog(1) << "Skipping over scheduling input node: " << node->func.name() << "\n";
             auto child = make_child();
             child->num_decisions_made++;
             accept_child(std::move(child));
             return;
         }
 
         if (!node->outgoing_edges.empty() && !root->calls(node)) {
-            aslog(0) << "In state:\n";
```

Comment:
This is all printing an elaborate error message, rather than verbose logging. I'd say to just revert to aslog(0), but the dump() won't trigger. Maybe dump should all be aslog(0) too, and  the other calls to dump() should be conditioned on the logging level?

Reply:
Ouch, yeah, I guess. Let me see what I can do.

Reply:
Better yet might be to pass an ostream (or similar) in to dump methods, and let the caller decide... but I'll save that for another day